src/app/components/Grid/index.jsx

```diff
@@ -7,7 +7,7 @@ import {
 } from '@bbc/gel-foundations/breakpoints';
 import { C_GHOST } from '@bbc/psammead-styles/colours';
 
-const group4WrapperMaxWidth = `63rem`; // 1008px
+export const group4WrapperMaxWidth = `63rem`; // 1008px
```

Comment:
As discussed [here](https://github.com/bbc/simorgh/pull/5698/files#r386262684), we are thinking to remove these constants and use the values directly from the `gel-foundations` package.

Reply:
Made this change.